src/locales/fi.json

```diff
@@ -88,6 +106,16 @@
   "home": {
     "title": "Venepaikka admin interface"
   },
+  "forms": {
+    "common": {
+      "edit": "Muokkaa",
+      "errors": {
```

Comment:
Just a note: displaying these errors change the modal's height which makes the user experience a bit "restless"